### DISCUSSION TITLE
Images not visible or not loading in readme.md and also in relative url of github

### DISCUSSION THREAD

**Discussion by MangalNathYadav** [community/community#53013] [Repositories] [answered]
### Select Topic Area

General

### Body

**i will not waste time will proceed using images.**

see in this there is no image loading
![image_2023-04-17_140428442](https://user-images.githubusercontent.com/68454356/232430529-1f196965-5f1b-4ce5-967e-f13822023607.png)
here's the code
![image_2023-04-17_140454518](https://user-images.githubusercontent.com/68454356/232430631-5d562b42-10e6-49fb-9fd3-1c7246699815.png)

here it is upload it also not visible there 
![image](https://user-images.githubusercontent.com/68454356/232430859-0ae01bef-146d-43cf-9486-0da6c90258b2.png)
![image](https://user-images.githubusercontent.com/68454356/232431385-33d90350-12c3-4ca6-a4fa-3bd49631dde9.png)

**Now my problem goes from here** 
This problem is same in my all repositories..  I don't know the cause if anyone know please tell me. 
you can check my repositories. I tried restarting my network services and used private window to see if there any problem in my cookies but its the same even i saw in private window..

**Comment by SMelidoni** [comment]
Hi @MangalNathYadav 👋🏽

You can try using relative paths instead of just the filename. Make sure the paths are correct, and the file extensions are accurate. Here's an example of how you can add images using relative paths:

```markdown
![Image 1](./site_img.jpg)
![Image 2](./site_img3.jpg)
![Image 3](./siteimg2.jpg)
```

**Reply by MangalNathYadav** [reply]
see it doesnt work i tried everything mostly...
in 2022 it was working all well then  this happened then it was working well in january 2023 but now it again started this problems comes evrytime when i use git. i couldnt figure out the root cause of this
![image](https://user-images.githubusercontent.com/68454356/232558983-49507916-d021-4075-a1c7-4c1863b25687.png)

![image](https://user-images.githubusercontent.com/68454356/232558931-315b722a-9f87-4864-96be-0126a8127446.png)
![image](https://user-images.githubusercontent.com/68454356/232559066-a438bf04-5cfe-4598-9317-f721cba1b2be.png)


and the same goes in my profle when i am using github this happens everytime...

![image](https://user-images.githubusercontent.com/68454356/232559239-3f2a243f-0f50-47e3-86d8-1a4dde432a50.png)

**Reply by SMelidoni** [reply]
Upon looking at the repository you showed in your response above, it seems like it is working! Is this working for you now?

![Screenshot 2023-04-17 at 18 39 08](https://user-images.githubusercontent.com/106830374/232566610-e898ede0-5b83-469e-bd77-8ed2fea1df20.png)

**Reply by MangalNathYadav** [reply]
Let me see

**Reply by MangalNathYadav** [reply]
It's not working for me ... 
In the above repository i changed the image type from png to jpg but still not working in my  and 
In afternoon my profile was clear and now this problem is in profile also...
![Screenshot_20230417-232942](https://user-images.githubusercontent.com/68454356/232573054-ec48b066-7592-454e-aa01-2c363a4c5765.jpg)
![Screenshot_20230417-233240](https://user-images.githubusercontent.com/68454356/232573091-679c71b1-af3a-4359-a730-98c41a57d219.jpg)
![Screenshot_20230417-233254](https://user-images.githubusercontent.com/68454356/232573118-ef927f00-eaf3-4679-b8dd-447ac25c7cf7.jpg)

**Reply by SMelidoni** [reply]
That's strange cause it's showing perfectly fine for me (I cleared my cache and also tested in an incognito browser) and it's showing as expected:

<img width="896" alt="Screenshot 2023-04-17 at 20 46 20" src="https://user-images.githubusercontent.com/106830374/232595014-d84bf1d4-a20e-41dc-b3af-12f7b0bd4a53.png">

How's your internet/WiFi? Could it possibly be a network issue rather than a GitHub one?

**Reply by MangalNathYadav** [reply]
![Screenshot_20230418-071257](https://user-images.githubusercontent.com/68454356/232648497-67d394c5-8797-4f95-ae0d-940d49034a58.jpg)
![Screenshot_20230418-065622](https://user-images.githubusercontent.com/68454356/232648517-b0721f79-fa63-42cf-8ae8-d14b2e580459.jpg)
![Screenshot_20230418-071332](https://user-images.githubusercontent.com/68454356/232648565-d7bf2a9d-4c04-4cd4-a03a-5248c7580fad.jpg)


Alright I see this problem is from my side...
When I connected to VPN it works well...
Without vpn images doesn't load
I restarted my network so many times 
It's Jio 5G
Same in incognito window (private window)
Vpn corrects this problem for me...
But any idea how can it be corrected without vpn

It doesn't make any sense after all..

**Reply by adityajadhav02** [reply]
Similar issue with Jio 4G. 
With mobile hotspot the images don't load. But when connected to VPN, it works fine.

**Reply by MangalNathYadav** [reply]
Did you find any solution for it?

**Reply by RockChinQ** [reply]
I meet the same problem in my profile https://github.com/RockChinQ/RockChinQ , is there any way to fix it?

**Reply by airtower-luna** [reply, marked as answer]
I've seen *a lot* of reports on this forum about Jio blocking `raw.githubusercontent.com` (where those images would load from) at the DNS level, see #32889 for example. That's why using a VPN (or just an alternative DNS resolver, e.g. [Quad9](https://quad9.net/)) makes it work. It may or may not help with other blocks, too, depending on how they are implemented.

**Reply by Suundumused** [reply]
It maybe github bug. I have the same problem, sometimes images loads or not.

**Reply by NotAGanesh** [reply]
i have an same problem thanks so what can i do in my region?

**Reply by sid760** [reply]
I was having the same problem, then I changed from default jio dns to google dns, and now it works fine. Thanks to @airtower-luna's [answer ](https://github.com/orgs/community/discussions/53013#discussioncomment-5871832) 👍 
I wonder what all other sites don't work.
btw I use jio fiber.

**Comment by StefanShivarov** [comment]
I just checked your repository and I can see the images on the readme file. Might be some issue on your end.

**Reply by MangalNathYadav** [reply]
I see

**Reply by saksham24** [reply]
On jio internet mobile/Fiber it doesn't load images but it works well with airtel 😆

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @MangalNathYadav ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by MangalNathYadav** [reply]
Thank you

**Reply by bHabawel** [reply]
Your repository img works well on my end though

**Comment by RockChinQ** [comment]
Hi there! I met this problem too, some of the img elements in the [README of my profile](https://github.com/RockChinQ/RockChinQ) repo cannot be loaded:

![image](https://github.com/community/community/assets/45992437/1052a679-403e-465c-acbf-4585b9d10efe)

while the browser debug page show errors below:

![image](https://github.com/community/community/assets/45992437/030a19dd-1d7e-4d36-89b7-1230537a82d8)

but I can directly visit these imgs in single pages:

![image](https://github.com/community/community/assets/45992437/cccc1b9f-3dc7-4db4-bd86-1b39046157a3)

and the [README of my friend's profile](https://github.com/Soulter) works well while he's using the img from the same host as me:

![image](https://github.com/community/community/assets/45992437/582a506c-3858-4641-8492-8d485274c132)

it's also works well in the GitHub Mobile app:

![(%9~F(1O8S%07M_N4MBHE(J](https://github.com/community/community/assets/45992437/77189c6a-c9a2-469f-aac5-6af87dc79566)

then I search web and found https://github.com/tomchentw/react-google-maps/issues/174 tells it's a issue of backend, so I reach here.
Thank you very much for your attention and waiting for your help!

**Comment by TextZip** [comment]
Hi, 
Images appear broken in my GitHub Profile as well. Even a simple line like this doesn't render. 
`<a href="mailto:textzip@gmail.com" target="blank"><img align="center" src="https://cdn.worldvectorlogo.com/logos/official-gmail-icon-2020-.svg" alt="jai.krishna.7982" height="30" width="40" /></a>` 

Some screenshots of the broken images:
![image](https://github.com/community/community/assets/49020643/5632e6f0-63e4-4d41-90f4-2d48a953f9f4)

I also use a [workflow](https://github.com/gautamkrishnar/blog-post-workflow) to periodically update my readme with new posts from my blog and videos from my YouTube. I recently noticed that the thumbnails for the YouTube Videos are broken. 

I initially raised an [issue](https://github.com/gautamkrishnar/blog-post-workflow/issues/216) at the workflow repo, but after some discussion realized that the issue probably had something to do with CSP rather than the workflow. 

The same workflow works for this [Profile](https://github.com/Dexters-Hub/Dexters-Hub/tree/master), so it isn't an issue with the workflow. 

Here are some more screenshots of the page errors when I inspect my profile page:
![image](https://github.com/community/community/assets/49020643/849ecf11-0355-4257-bc4b-9449034d70a0)
![image](https://github.com/community/community/assets/49020643/d162527d-9cc6-41ee-9889-4026427abcb5)

The [initial issue](https://github.com/gautamkrishnar/blog-post-workflow/issues/216) I raised has more details and context. I didn't want to reiterate everything from there here. 

Any help in resolving this would be really great, TIA

**Comment by startup-dreamer** [comment]
All of my project's readme images have disappeared. 
![image](https://github.com/community/community/assets/106421807/48e18aed-5bde-4d0f-bfe0-2d6913da80d5)
 Until yesterday, they were working fine. Some of the projects that I was contributing to also have missing images in their readmes. 
![image](https://github.com/community/community/assets/106421807/6dd111e1-97bb-49c5-981b-83c18c5640a0)


When I open the repository, the console suggests this error:
![image](https://github.com/community/community/assets/106421807/b5fef8cf-52cb-49bc-94aa-3e405a493f87)

Any way to resolve this.

**Comment by MangalNathYadav** [comment]
Use VPN and then check

**Reply by startup-dreamer** [reply]
got it I guess the problem was from my institutes WIFI

**Comment by MangalNathYadav** [comment]
its the same issue with my Jio network

**Reply by a-y-u-s-h** [reply]
How did you fix it?

**Reply by MangalNathYadav** [reply]
Use vpn

**Reply by shvamabps** [reply]
That's the final solution but why can't it be solved on the original network instead using a vpn to solve the issue.

**Reply by MangalNathYadav** [reply]
never know, official got no answer for this. till then use vpn

**Reply by sid760** [reply]
You DON'T need to use vpn, you can just change **dns**, to any other provider, in your device, and it will work!
For example I am using the google's public dns service, and there are many other dns server, you can use anyone, and don't have to stick to jio's default dns.

**Reply by sid760** [reply]
No you got the answer here: https://github.com/orgs/community/discussions/53013#discussioncomment-5871832

**Comment by pratyushgguptaa** [comment]
Yeah facing same issue here. Does not load up when connected to any JIO network. On other networks, it works just fine. It should be properly fixed.

**Comment by sid760** [comment]
I guess your issue is answered, I just want to know what browser **theme** are you using?? it looks so cool!!
Thanks
